### ISSUE TITLE
[Experiment] The iOS Notification opt-in card is not displayed on the first-run onboarding

### ISSUE THREAD

**Issue by Softvision-PatriciuPop** [mozilla-mobile/firefox-ios#13811]
## Prerequisites
- Have the Firefox Release 112.0 (29756) TestFlight build installed.
## Steps to reproduce
1. Open the browser with the profile from prerequisites.
2. Tap the "Get Started" button from the "Welcome to an independent internet" card.
3. Tap the "Skip" button from the "Hop from phone to laptop and back" card and observe the behavior.
### Expected behavior
- The "Notifications help you do more with Firefox" card is displayed in the onboarding process.
### Actual behavior
- The Notification opt-in card is not part of the onboarding and the Homepage is displayed.
### Device & build information
* Devices: 
  - iPhone 11 (iOS 16.2)
  - iPhone 13 mini (iOS 15.6)
* Build version: Release 112.0 (29756)
### Notes
- The issue is reproducible using both the "treatment-a" and "treatment-b" branches of the experiment.
- The "Firefox would like to send you notifications" pop-up is displayed after the onboarding flow on all branches of the experiment.
## Attachments: 

https://user-images.githubusercontent.com/40393891/229071527-1d8147ed-4190-46a4-a8b0-fc7e30c11d7f.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6104)

**Comment by dnarcese** [comment]
@Softvision-PatriciuPop do you remember which branch you were initially enrolled in? If you change the branch, and then re-install the app to show onboarding, I don't think the branch enrolment would save. 
We are adding back the 'Show Tour' option in settings so that you can enrol in a branch and then trigger onboarding.

But, if you were already in a treatment branch then this would be a bug.

**Comment by dnarcese** [comment]
When build 112.1 (29908) is out, it should have the 'Show Tour' option in the settings again. This way you are able to trigger onboarding manually.

**Comment by Softvision-PatriciuPop** [comment]
The behavior mentioned above was while enrolled in the treatment-a and treatment-b branches of the experiment and I can confirm that first-run enrollment is working as intended.  I've retested the issue today and it's no longer reproducible using the latest build, the notification card is correctly displayed on both branches. Tested using Firefox Release 112.1 (29908) on iPhone 11 (iOS 16.2). Marking this as resolved.
